pkg/api/tracingapi/client_kprobe.go

```diff
@@ -262,6 +262,11 @@ type MsgGenericKprobeSockaddr struct {
 	SinAddr   [2]uint64
 }
 
+type MsgGenericKprobeSockaddrUn struct {
+	Family uint16
```

Comment:
For sockaddr_un, Family is always AF_UNIX (according to the man page). Do we need to record it? In inet we do because it tells us whether we're IPv4 or IPv6 (or potentially something we don't understand).